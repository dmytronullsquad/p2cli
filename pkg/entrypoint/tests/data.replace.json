{
  "value": "This will have the spaces removed",
  "match": "This",
  "replace": "Hello",
  "count": 1
}