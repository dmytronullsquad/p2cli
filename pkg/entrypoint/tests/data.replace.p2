String Replace
{{ value | replace:[" ", ""] }}

String Replace with count
{{ value | replace:[" ", "", 1] }}