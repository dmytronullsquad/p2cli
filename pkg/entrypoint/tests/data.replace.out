String Replace
Thiswillhavethespacesremoved

String Replace with count
Thiswill have the spaces removed